lib/index.js

```diff
@@ -117,3 +99,21 @@ export default class Roots extends EventEmitter {
 function npmInstall (opts) {
   return node.call(exec, 'npm install', { cwd: opts.root })
 }
+
+function compileCallback (id, err, stats) {
+  if (err) {
+    return this.emit('error', new RootsError({ id: id, message: err }))
+  }
+
+  // Webpack "soft errors" are classified as warnings in roots. An error is
+  // an error. If it doesn't break the build, it's a warning.
+  const jsonStats = stats.toJson()
+  if (jsonStats.errors.length > 0) {
```

Comment:
Honestly I'm wondering why we need to put them to json in the first place, this was originally pulled from an example somewhere I think. But this isn't new code, it was just moved into a function.